<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myWidget</class>
 <widget class="QWidget" name="myWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1475</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>601</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>XimeaWindow</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>dt_zfloat_frame</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>color_frame</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>t_color_frame</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>580</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>ct_zfloat_frame</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>info</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_12">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_12">
      <property name="title">
       <string>policyView</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>pointNum:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>xi_score:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>dk_score:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>all_score:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>gradient:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>len_gradient:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>172</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>startDK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>143</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>saveXimea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>114</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>startXimea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>201</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>saveDK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>240</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>saveAll</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>360</y>
     <width>821</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>ur_control</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="2" column="4" rowspan="2" colspan="3">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="title">
       <string>tcp_move</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>X+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_14">
         <property name="text">
          <string>Y+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_16">
         <property name="text">
          <string>Z+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string>X-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_15">
         <property name="text">
          <string>Y-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_17">
         <property name="text">
          <string>Z-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_18">
         <property name="text">
          <string>Rx+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_20">
         <property name="text">
          <string>Ry+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_22">
         <property name="text">
          <string>Rz+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_19">
         <property name="text">
          <string>Rx-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_21">
         <property name="text">
          <string>Ry-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_23">
         <property name="text">
          <string>Rz-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="4" colspan="3">
     <widget class="QGroupBox" name="groupBox_9">
      <property name="title">
       <string>joint_move</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_24">
         <property name="text">
          <string>J1+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_25">
         <property name="text">
          <string>J2+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_26">
         <property name="text">
          <string>J3+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_27">
         <property name="text">
          <string>J1-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_28">
         <property name="text">
          <string>J2-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_29">
         <property name="text">
          <string>J3-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_30">
         <property name="text">
          <string>J4+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_31">
         <property name="text">
          <string>J5+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_32">
         <property name="text">
          <string>J6+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_33">
         <property name="text">
          <string>J4-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_34">
         <property name="text">
          <string>J5-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_35">
         <property name="text">
          <string>J6-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>169.254.220.1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>30003</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>getData</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>script:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>sendScript</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string>stopRightNow</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="pushButton_11">
      <property name="text">
       <string>saveData</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="2" column="0" rowspan="3" colspan="4">
     <widget class="QTextBrowser" name="textBrowser_2"/>
    </item>
    <item row="4" column="7">
     <widget class="QGroupBox" name="groupBox_11">
      <property name="title">
       <string>policy</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_38">
         <property name="text">
          <string>gradientMove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_41">
         <property name="text">
          <string>pause/continue</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_44">
         <property name="text">
          <string>adaptiveCapture</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="7" rowspan="4">
     <widget class="QGroupBox" name="groupBox_10">
      <property name="title">
       <string>memory</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_40">
         <property name="text">
          <string>setOrig</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_37">
         <property name="text">
          <string>returnToMem</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_43">
         <property name="text">
          <string>resetMem</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_36">
         <property name="text">
          <string>memorize</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_39">
         <property name="text">
          <string>returnToOrig</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton_45">
         <property name="text">
          <string>autoSaveOfMem</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_42">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>200</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>savePC</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
